src/js/components/awardv2/idv/activity/chart/ActivityChart.jsx

```diff
@@ -83,11 +84,20 @@ export default class ActivityChart extends React.Component {
         yRange.push(maxValueY);
         xRange.push(minValueX);
         xRange.push(maxValueX);
+        return { xRange, yRange };
+    }
 
+    graphWidthAndHeight() {
         // calculate what the visible area of the chart itself will be (excluding the axes and their
         // labels)
         const graphWidth = this.props.width - this.props.padding.left;
         const graphHeight = this.props.height - this.props.padding.bottom;
+        return { graphWidth, graphHeight };
+    }
+
+    generateChart() {
```

Comment:
@jonhill13 seems like it would make more sense possibly leave the `xyRange()` and `graphWidthAndHeight` logic inside this fn and abstract the work being done on `line 115` into a new fn? Or just add another `fn` invoked on `line 115` called `generateBar` like this:

### Line 115:
```javascript
const bars = this.props.awards.map((data, index) => {
  return generateBar(data);
})
```